websites/S/SoundCloud/presence.ts

```diff
@@ -154,97 +177,106 @@ presence.on("UpdateData", async () => {
           "#app > div.playControls.g-z-index-control-bar.m-visible > section > div > div.playControls__elements > div.playControls__soundBadge > div > div.playbackSoundBadge__titleContextContainer > div > a"
         )
         .getAttribute("href");
-    data.startTimestamp = startTimestamp;
-    data.endTimestamp = endTimestamp;
-    data.smallImageKey = playing ? "play" : "pause";
-    data.smallImageText = (await strings)[playing ? "play" : "pause"];
-    data.buttons = [
+
+    presenceData.startTimestamp = startTimestamp;
+    presenceData.endTimestamp = endTimestamp;
+
+    if (cover) {
+      presenceData.largeImageKey = document
+        .querySelector<HTMLSpanElement>(
+          ".playbackSoundBadge__avatar.sc-media-image > div > span"
+        )
+        .style.backgroundImage.match(/"(.*)"/)[1]
```

Comment:
Actually, it could be null

Reply:
Actually, it cannot be null, this is inside of the if case which checks whether something is playing or not, which means the bar at the bottom of the screen is shown.

Reply:
*Some don't have cover*

Reply:
got an example?

Reply:
https://soundcloud.com/user48301249/vvkwvzuw2psc?in=user48301249/sets/heart